pkg/dom0-ztools/Dockerfile

```diff
@@ -1,8 +1,12 @@
 FROM alpine:3.12 as zfs
+
 RUN mkdir -p /out/etc/apk /out/boot && cp -r /etc/apk/* /out/etc/apk/
 RUN apk add --no-cache --initdb -p /out \
     zfs=0.8.4-r0
 
+
 FROM scratch
 COPY --from=zfs /out/ /
-ADD rootfs/ /
+ADD rootfs.tar /out/
```

Comment:
It seems to me that this rootfs.tar is making things more complicated -- unless there's a good reason -- I'd rather avoid doing this and revert the rest of the changes to dom0-ztools

Reply:
Dropped the commit